WorldCat Discovery Service.js

```diff
@@ -298,14 +298,15 @@ var testCases = [
 					}
 				],
 				"date": "2010",
-				"ISBN": "9781598565836\n1598565834",
+				"ISBN": "9781598565836",
 				"language": "English",
 				"libraryCatalog": "WorldCat Discovery Service",
 				"numPages": "xx, 404",
 				"place": "Peabody, Mass.",
 				"publisher": "Hendrickson Publishers Marketing",
 				"series": "Lexham Bible reference series; Lexham Bible reference series.",
 				"shortTitle": "Discourse grammar of the Greek New Testament",
+				"url": "Table of contents http://bvbr.bib-bvb.de:8991/F?func=service&doc_library=BVB01&local_base=BVB01&doc_number=024386228&line_number=0001&func_code=DB_RECORDS&service_type=MEDIA",
```

Comment:
Not liking this could we do something like
1. Check if URL is actually a URL
2 a) if it is, then import it into URL field
2 b) if it is not, check if it's a ToC and then turn it into attached link with appropriate title?